code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -122,6 +160,13 @@
 				break
 	update_icon()
 
+/**
+ * setter for pipe layers
+ *
+ * Set the layer of the pipe that the device has to a new_layer
+ * Arguments:
+ * * new_layer - the layer at which we want the piping_layer to be (1 to 5)
+ */
```

Comment:
Minor point, we don't really use getter/setter too much, especially getter, you could prob get away with something like gets/sets here, up to you tho.

Reply:
what you mean? The proc is a setter... unless i'm mistaken on what a setter should be, i mean the name of the proc is `setPipingLayer` and what it does is set the piping layer to what we want

Reply:
ah I see what you're saying, ignore me